gammapy/data/observations.py

```diff
@@ -68,6 +68,29 @@ class ObservationCTA(object):
     meta : `~collections.OrderedDict`
         Dictionary to store metadata
 
+    Examples
+    --------
+    Create an observation from the CTA 1DC:
+
+    >>> from gammapy.data import ObservationCTA, EventList, GTI
+    >>> from gammapy.irf import EnergyDependentMultiGaussPSF, EffectiveAreaTable2D, EnergyDispersion2D, Background3D
+    >>>
+    >>> data_file = '$GAMMAPY_EXTRA/datasets/cta-1dc/data/baseline/gps/gps_baseline_110380.fits'
+    >>> cal_file = '$GAMMAPY_EXTRA/datasets/cta-1dc/caldb/data/cta/1dc/bcf/South_z20_50h/irf_file.fits'
+    >>>
+    >>> kwargs = dict(events = EventList.read(data_file),
+    >>>               gti = GTI.read(data_file),
+    >>>               psf = EnergyDependentMultiGaussPSF.read(cal_file, hdu='Point Spread Function'),
+    >>>               aeff = EffectiveAreaTable2D.read(cal_file),
+    >>>               edisp = EnergyDispersion2D.read(cal_file, hdu='Energy Dispersion'),
+    >>>               bkg = Background3D.read(cal_file),
+    >>>               pointing_radec = EventList.read(data_file).pointing_radec,
+    >>>               observation_live_time_duration = EventList.read(data_file).observation_live_time_duration,
```

Comment:
You shouldn't read the event list multiple times.
Suggest to read one thing at a time, create local variables and put the call at the end.

Like this:
```
events = EventList.read(...)
...
obs = ObservationCTA(
    events=events,
    ...
    observation_live_time_duration=events.observation_live_time_duration,
)
```

Reply:
Done